text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,129 @@
+---
+Title: Transitioning from X/Twitter to new social media platforms
+Drivers: Théo Zimmermann (@Zimmi48)
+---
+
+## Summary
+
+This RFC proposes to reform the management of the official social media channels of Coq in the context of the renaming to the Rocq Prover. In particular, it proposes to: search for additional volunteers to help manage them, and to revisit which social media platforms to be present on in the current context.
+
+## Context and motivation
+
+During the last four years, the Coq project's social media presence has been limited to the CoqLang account on Twitter (then X). This account was initially managed by three persons: Emilio Jesús Gallego Arias, Anton Trunov, and Théo Zimmermann. However, in the last two years, the account has been mostly managed by Emilio Jesús Gallego Arias alone (Anton Trunov is not active anymore in the Coq ecosystem, and Théo Zimmermann has stopped using X/Twitter).
+
+Leaving the management of the account on the shoulders of a single person is not sustainable in the long term. If the project wants to maintain a social media presence, it needs to involve more people in the management of its social media account(s).
+
+The CoqLang account uses the Coq-community logo instead of the Coq logo, because it is largely considered as a better visual identity for the Coq project. However, the CoqLang account is still the official account of the Coq project. In the context of the renaming of Coq to the Rocq Prover, we have to decide what to do with respect to our social media presence. If we keep the CoqLang account, we will have to rename it and to update its visual identity to use the new Rocq Prover logo.
+
+With the change of ownership of X/Twitter and the associated changes in moderation policies, the general social media context is much different than it was four years ago when it was decided to create the CoqLang account. In particular, many academics have left X/Twitter and moved to alternative platforms such as Mastodon (or more recently Bluesky). More recently, many journalists have also started to leave X/Twitter. Beyond individual decisions, there are now media (such as [The Guardian][guardian] in the UK, [La Vanguardia][lavanguardia] in Spain, [NPR][npr] in the US, [Ouest France][ouestfrance] and [Mediapart][mediapart] in France) and academic institutions (e.g., [Aix-Marseille University][amu], [University of Strasbourg][unistra], [University of Luxembourg][unilux], [RWTH Aachen][rwth], [Utrecht University][uu], [York University][york]) that are officially leaving X/Twitter as well.
```

Comment:
Do we have a sense how broad the movement away from Twitter is?  Press attention can create misleading impressions about trends (e.g. reports of a few burglaries can get blown up into a crime wave).  If it's only a few percent of academic institutions, is it really necessary and desirable for Rocq to follow suit?

Another thing I'd consider is whether the most prestigious institutions are withdrawing from Twitter.  I briefly checked on a couple of these but didn't see evidence that any had left Twitter.

BTW, it looks like Inria still uses Twitter.  https://www.inria.fr/fr/adresses-des-sites-inria still has a button for sharing things on Twitter.

Reply:
I was using these examples to show that the movement away from Twitter is not limited to individuals. It is expected that institutions will not be as quick as individuals to leave. But our target is not institutions but individual followers. And those have indeed departed Twitter en-masse. While I was searching for reference, I even saw an academic paper measuring the effect (cannot find it again right now). The departures from Twitter are hard to measure though because most people keep their account (they just become silent). That's my case, for instance. But what can be measured is the growth of alternative platforms such as Mastodon and Bluesky, and this has been huge. Personally, I moved away two years ago to Mastodon, and I have no reason to look back given the size of my academic network over there on Mastodon. It includes many people from the Coq/Rocq community.